vendor/wp-now/src/wp-now.ts

```diff
@@ -565,18 +566,6 @@ set_error_handler(function($severity, $message, $file, $line) {
 	`
 	);
 
-	php.writeFile(
-		path.posix.join( PLAYGROUND_INTERNAL_MU_PLUGINS_FOLDER, '0-dns-functions.php' ),
-		`<?php
-		// Polyfill for DNS features which are not currently supported by @php-wasm/node.
-		// See https://github.com/WordPress/wordpress-playground/issues/1042
-		// These specific features are polyfilled so the Jetpack plugin loads correctly, but others should be added as needed.
-		if ( ! defined( 'DNS_NS' ) ) {
```

Comment:
It was added in Playground, correct?

Reply:
Yes, we added it in Playground. I also tested it locally to make sure.